### ISSUE TITLE
Chyby zobrazení oznámení

### ISSUE THREAD

**Issue by motyc** [ARUP-CAS/aiscr-webamcr#526]
https://wamcr-test.aiscr.cz/arch-z/akce/detail/C-200913623C
- [ ] Při odpojení PIAN pomocí odpovídající funkce vyskočí dvě hlášky, které se ale překrývají:
![image](https://user-images.githubusercontent.com/20186892/171847201-24b8d7c2-846d-487a-b8da-e6697df24b6d.png)

https://wamcr-test.aiscr.cz/arch-z/akce/detail/C-201227819A
- [ ] po úpravě DJ se oznámení schovává pod mapu:
![image](https://user-images.githubusercontent.com/20186892/173638793-d135ba51-1561-4b59-8457-d6a2344540e1.png)

**Comment by pesikj** [comment]
@motyc @ScottVanWagenen 
Souhlasím, že jde o bug.

**Comment by motyc** [comment]
### Výsledek testu
- [ ] ve druhém případě se již neschovává pod mapou, ale oznámení jen problikne a hned zmizí (asi v důsledku reloadu mapy)

**Comment by jnihnat** [comment]
muzete prosim jeste jendou skusit, pripadne presneji specifikovat pri uprave ceho se to takto chova?
Mne se to nepovedlo nasimulovat totiz.

**Comment by motyc** [comment]
@jnihnat Souhlasím, také se mi to teď nedaří nasimulovat. Berme jako v pořádku.
